### ISSUE TITLE
When you are in an app, search should only search that app

### ISSUE THREAD

**Issue by warpech** [StarcounterSamples/Launcher#167]
When you are in an app, search should only search that app. Like on GitHub:

<img width="379" alt="screenshot 2015-11-20 11 23 02" src="https://cloud.githubusercontent.com/assets/566463/11297724/a5c6e508-8f7a-11e5-83ef-e7266e85924e.png">

@jwillmer proposes that the "Search" placeholder could be renamable from the app.

**Comment by jwillmer** [comment]
Yes, renamable to give the user an idea what the search result could be, like a placeholder with watermark: "Products".
